docs/core/extensions/options.md

```diff
@@ -33,12 +33,12 @@ When using the options pattern, an options class:
 - Must be non-abstract with a public parameterless constructor
 - Contain public read-write properties to bind (fields are ***not*** bound)
 
-The following code:
+The following code is part of the _Program.cs_ C# file and:
 
 * Calls [ConfigurationBinder.Bind](xref:Microsoft.Extensions.Configuration.ConfigurationBinder.Bind%2A) to bind the `TransientFaultHandlingOptions` class to the `"TransientFaultHandlingOptions"` section.
-* Displays the configuration data.
+* Displays the configuration data when ran.
```

Comment:
Should this be:
```suggestion
* Displays the configuration data when run.
```
Or should it even be there are all? I mean isn't it assumed it nothing is done until its run?